<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string extracomment="Amazon Game">亚马逊棋</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnPrevStep">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>160</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>悔棋</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNextStep">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>210</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>后移</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMsg">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>0</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;对弈信息&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClearMsg">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>540</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>清空信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStopAI">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>260</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>终止AI计算</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRow">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0                 1                 2               3                 4                5                6               7                8                9</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRow_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>530</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0                 1                 2               3                 4                5                6               7                8                9</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>340</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>440</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>140</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>240</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>290</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>390</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>490</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>游戏</string>
    </property>
    <addaction name="actionBegin"/>
    <addaction name="actionEndGame"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>设置</string>
    </property>
    <widget class="QMenu" name="menuPVPOrder">
     <property name="title">
      <string>人人模式</string>
     </property>
     <addaction name="actionBlackFirst"/>
     <addaction name="actionWhiteFirst"/>
    </widget>
    <widget class="QMenu" name="menuPVCOrder">
     <property name="title">
      <string>人机模式</string>
     </property>
     <addaction name="actionPersonFirst"/>
     <addaction name="actionComputerFirst"/>
    </widget>
    <widget class="QMenu" name="menuAI">
     <property name="title">
      <string>AI引擎选择</string>
     </property>
     <addaction name="actionTestEngine"/>
     <addaction name="actionMCRave"/>
    </widget>
    <addaction name="menuPVPOrder"/>
    <addaction name="menuPVCOrder"/>
    <addaction name="actionOnline"/>
    <addaction name="separator"/>
    <addaction name="menuAI"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAboutThis"/>
    <addaction name="actionAboutAuthor"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuSetting"/>
   <addaction name="menuInfo"/>
  </widget>
  <action name="actionBegin">
   <property name="text">
    <string>开始</string>
   </property>
  </action>
  <action name="actionEndGame">
   <property name="text">
    <string>结束本局</string>
   </property>
  </action>
  <action name="actionWhiteFirst">
   <property name="text">
    <string>白方先手</string>
   </property>
  </action>
  <action name="actionPersonFirst">
   <property name="text">
    <string>人先手</string>
   </property>
  </action>
  <action name="actionComputerFirst">
   <property name="text">
    <string>计算机先手</string>
   </property>
  </action>
  <action name="actionOnline">
   <property name="text">
    <string>联网模式</string>
   </property>
  </action>
  <action name="actionAboutThis">
   <property name="text">
    <string>关于软件</string>
   </property>
  </action>
  <action name="actionAboutAuthor">
   <property name="text">
    <string>关于作者</string>
   </property>
  </action>
  <action name="actionBlackFirst">
   <property name="text">
    <string>黑方先手</string>
   </property>
  </action>
  <action name="actionTestEngine">
   <property name="text">
    <string>TestEngine</string>
   </property>
  </action>
  <action name="actionMCRave">
   <property name="text">
    <string>MCRave</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
